recipes/mypy_extensions/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mypy_extensions" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "c4553f2475996c912658d3c2f398766cd47a65e0c2c6bb27ac8dea94a21640f4"
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - typing  # [py<35]
```

Comment:
Does this require `mypy` as well?

Reply:
Seeing the source code at https://github.com/python/mypy/blob/master/extensions/mypy_extensions.py doesn't have any reference to mypy.